internal/kube/deployment.go

```diff
@@ -6,8 +6,19 @@ import (
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
+
+	. "k8s.io/apimachinery/pkg/api/errors"
 )
 
+func newClient() (kubernetes.Interface, error) {
+	kubeConfig, err := clientcmd.BuildConfigFromFlags("", clientcmd.RecommendedHomeFile)
+	if err != nil {
+		return nil, err
+	}
+
+	return kubernetes.NewForConfig(kubeConfig)
+}
```

Comment:
This function has no parameter and it returns really important configuration(kubeconfig)
I think this could be used in other lines of code at some time.
How about moving this function to other shared file?
```
client, err := config.New()
if err != nil {
    return err
}

# in config.go
func New() (kubernetes.Interface, error) {
	kubeConfig, err := clientcmd.BuildConfigFromFlags("", clientcmd.RecommendedHomeFile)
	if err != nil {
		return nil, err
	}

	return kubernetes.NewForConfig(kubeConfig)
}
```

Reply:
It's good points. I've updated it. https://github.com/DevopsArtFactory/klocust/pull/8/commits/c72fafa03806cab4d3a96c8b42bdccbe4ef06c6a